terminus-ui/toggle/src/toggle.component.html

```diff
@@ -8,6 +8,7 @@
   [disabled]="isDisabled"
   [name]="name"
   (change)="change.emit($event)"
+  (change)="selectionChange.emit($event)"
```

Comment:
I honestly had no clue we could use the same emitter twice on an element 🙃 learn something new every day!